Descirption: disable building codec_ilbc
Author: Tzafrir Cohen <tzafrir@debian.org>
Forwarded-Upstream: no
Bug-Debian:

As we have to strip the ilbc code from asterisk, we need to disable
building codec_ilbc and cleaning the ilbc/ directory.

Patch needs to be cleaned-up to be uploaded upstream.

--- a/codecs/codec_ilbc.c
+++ b/codecs/codec_ilbc.c
@@ -26,6 +26,7 @@
  */
 
 /*** MODULEINFO
+	<defaultenabled>no</defaultenabled>
 	<support_level>core</support_level>
  ***/
 
--- a/codecs/Makefile
+++ b/codecs/Makefile
@@ -27,7 +27,6 @@ LIBLPC10 := $(SUB_LPC10)/liblpc10.a
 
 SUB_DIRS := \
 	$(SUB_GSM) \
-	$(SUB_ILBC) \
 	$(SUB_LPC10) \
 
 all: _all
